src/main.cpp

```diff
@@ -49,9 +49,15 @@ int main( int argc, char *argv[]) {
 
     if(!mq.is_only_instance()) {
         std::string msg;
+        std::string cmd;
+
         if(argc == 1) {
-            mq.SendMessage("kill");
-        } else if(static_cast<std::string>(argv[1]) == "help") {
+            cmd = "kill";
+        } else {
+            cmd = static_cast<std::string>(argv[1]);
```

Comment:
Is there any reason to use `static_cast<std::string>(argv[1]) `over `std::string(argv[1])`?
If I'm not mistaken, it should do the exact same thing.